### PR TITLE
Kim's Week 1 Quiz

### PR DESCRIPTION
**Pull request by kimcabb**
I had trouble getting the tags I wrote in my answers to display in the md file, so I created a txt file, too.

### CODE REVIEW

**Comment by mattscilipoti** [comment]
In the future, please put your answer between the formatting directives (triple tics, ````text`).  Just replace the text that says "Your answer here", not the directive around it.  That is why you could not see your html tags.

**Comment by kimcabb** [comment]
Okay, Matt. That's very good to know, thank you.

Kim

File: README.md [hunk 1]
```diff
@@ -15,9 +15,13 @@ the square brackets (for multiple-choice questions).
...
-```text
-Your answer here.
-```
+Semantic tags are HTML tags whose names clearly describe their content.  These tags help the developer separate content structure from styling.  For example, the "<footer>" tag defines footer content:
+
+	<footer>This content goes in the footer.</footer>
+
+Non-semantic tags do not describe their content and sometimes focus on content styling.  An example is the <b></b> tag, which is used to create the bold text:
+
+	<footer><b>This footer content is now bold.</b></footer>
```
Comment (mattscilipoti) [inline README.md:24]: :+1:

File: README.md [hunk 2]
```diff
@@ -27,17 +31,13 @@ What is the purpose of the `alt` attribute? e.g:
 <img src='https://jesse.sh/img/me.jpg' alt='Jesse Shawl'/>
 ```
 
-```text
-Your answer here.
-```
+The 'alt' attribute provides alternative text in the event the associated image cannot load in the page.  In the above example, "Jesse Shawl" would display in place of the image.
```
Comment (mattscilipoti) [inline README.md:34]: :+1:

File: README.md [hunk 3]
```diff
@@ -27,17 +31,13 @@ What is the purpose of the `alt` attribute? e.g:
...
 
 ### Question #3
 
 What is the purpose of the `<head></head>` tag in HTML?
 
-```text
-Your answer here.
-```
+The <head></head> tag is used to define page content that is not displayed, necessarily, but used to provide information about the document (metadata, for example).  It should be placed before the <body></body> tag and will contain other tags such as <title></title>; <link></link> to reference an external stylesheet (or <style></style> for internal styling); and/or <script></script> to call javascript functions, for example.
```
Comment (mattscilipoti) [inline README.md:40]: :+1:

File: README.md [hunk 4]
```diff
@@ -49,36 +49,31 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
...
-```text
-___: Defines the distance between an element's border and adjacent elements' borders.
-___: Inserts a "wall" around an element.
-___: Defines the distance between an element's content and its border.
+MARGIN: Defines the distance between an element's border and adjacent elements' borders.
+BORDER: Inserts a "wall" around an element.
+PADDING: Defines the distance between an element's content and its border.
 ___: Defines the width of an element.
-___: Overlays a "wall" on top of an element.
+OUTLINE: Overlays a "wall" on top of an element.
```
Comment (mattscilipoti) [inline README.md:56]: :+1:

File: README.md [hunk 5]
```diff
@@ -49,36 +49,31 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
...
 ### Question #5
 
 What does the following selector do?  `ul.dropdown > li`?
 
 Select 1:
-```
+
 [] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[X] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
```
Comment (mattscilipoti) [inline README.md:66]: Close.  What's your second guess?

File: README.md [hunk 6]
```diff
@@ -49,36 +49,31 @@ The choices are `border`, `outline`, `padding`, and `margin`. You will leave two
...
 Identify the three places CSS can go, and rank them in terms of specificity:
 
-```text
-1.
-2.
-3.
-```
+1. Inline
+2. Internal
+3. External
```
Comment (mattscilipoti) [inline README.md:76]: :+1:

File: README.md [hunk 7]
```diff
@@ -88,25 +83,20 @@ Which of the following represents a correct workflow for submitting a PR on a no
 (ignore the lack of commit messages)
 
 Select 1:
-```
-[] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+
+[X] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
```
Comment (mattscilipoti) [inline README.md:87]: :+1:

File: README.md [hunk 8]
```diff
@@ -88,25 +83,20 @@ Which of the following represents a correct workflow for submitting a PR on a no
...
-```
 
 ### Question 8
 
 What is the difference between a fork and a clone?
 
-```text
-Your answer here.
-```
+In Git, a fork is a repository copy that is stored in one's remote repository.  A clone is a repository copy that is stored in one's local repository.
```
Comment (mattscilipoti) [inline README.md:96]: :+1:

File: README.md [hunk 9]
```diff
@@ -88,25 +83,20 @@ Which of the following represents a correct workflow for submitting a PR on a no
...
 
 ### Question 9
 
 How is `git pull` related to `git fetch`?
 
-```text
-Your answer here.
-```
+Git 'pull' and git 'fetch' both grab current copies of a remote branch's repository and store them locally.  However, git 'pull' goes one step further and merges changes made to the current branch into the local copy.
```
Comment (mattscilipoti) [inline README.md:102]: :+1:

**Comment by mattscilipoti** [comment]
8/9. Good job. You can find our recommendations in the solution branch.

**Pull request closed without merging**